ui/src/main/java/com/doordeck/sdk/common/utils/UuidUtils.kt

```diff
@@ -0,0 +1,14 @@
+package com.doordeck.sdk.common.utils
+
+import java.util.UUID
+
+object UuidUtils {
```

Comment:
does it even need to be an object?

Reply:
Using a Kotlin `object` can group better utility functions under the same purpose (UUID). There is only one so it's scalable from my point of view.

Kotlin users also prefer to use utility functions as top level (the way you propose it) if they have a single purpose and won't have a purpose of grouping.

So for me it's only a scalable situation where we can add more `UUID` utils. I don't see many more, so I'll put it at top level